docs/lang/articles/debug/debugging.md

```diff
@@ -116,20 +79,15 @@ a = ti.field(ti.f32, 4)
 def foo():
     a[0] = 1.0
     print('a[0] = ', a[0]) # right
-    print(f'a[0] = {a[0]}') # wrong, f-string is not supported
-    print("a[0] = %f" % a[0]) # wrong, formatted string is not supported
+    print(f'a[0] = {a[0]}') # wrong: f-strings are not supported
+    print("a[0] = %f" % a[0]) # wrong: formatted strings are not supported
 
 foo()
 ```
 
-:::
-
 ## Compile-time `ti.static_print`
 
-Sometimes it is useful to print Python-scope objects and constants like
-data types or SNodes in Taichi-scope. So, similar to `ti.static`, Taichi
-provides `ti.static_print` to print compile-time constants, which is similar
-to Python-scope `print`:
+It can be useful to print Python objects and their properties like data types or SNodes in the Taichi scope. Based on `ti.static` (see [Metaprogramming](../advanced/meta.md)), Taichi provides `ti.static_print` to print compile-time constants in the Taichi scope:
```

Comment:
```suggestion
It can be useful to print Python objects and their properties like data types or SNodes in the Taichi scope. Similar to `ti.static`, which makes the compiler evaluate an argument at compile time (see the [Metaprogramming](../advanced/meta.md) for more information), `ti.static_print` prints compile-time constants in the Taichi scope:
```